### PR TITLE
[WOR-167] Add null check

### PR DESCRIPTION
**Pull request by aherbst-broad**
## Context
There are instances where the `billingAccountName` field on a `ProjectBillingInfo` instance may be null, specifically when removing a billing account.

## This PR
* Adds a null check on the field before attempting to set the attribute on opencensus span

------

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check that the **Product Owner** has signed off on any user-facing changes
- [ ] **Submitter**: Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Database checks:
  * If PR includes new or changed db queries, include the explain plans in the description
  * Make sure liquibase is updated if appropriate
  * If doing a migration, take a backup of the
  [dev](https://console.cloud.google.com/sql/instances/terraform-qfarbdq3lrexxck5htofjs5z6m/backups?project=broad-dsde-dev&organizationId=548622027621)
  and
  [alpha](https://console.cloud.google.com/sql/instances/terraform-r4caezzc35c4tb7pgdhwkmme4y/backups?project=broad-dsde-alpha&organizationId=548622027621)
  DBs in Google Cloud Console
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] Tell your tech lead (TL) that the PR exists if they want to look at it
- [ ] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via Slack and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Review by marctalbott — APPROVED** [review]

File: core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala [hunk 1]
```diff
@@ -721,7 +721,9 @@ class HttpGoogleServicesDAO(
         blocking {
           val span = startSpanWithParent("executeGoogleRequest", s)
           span.putAttribute("googleProjectId", AttributeValue.stringAttributeValue(googleProjectId.value))
-          span.putAttribute("billingAccount", AttributeValue.stringAttributeValue(projectBillingInfo.getBillingAccountName))
+          if (projectBillingInfo.getBillingAccountName != null) {
```
Comment (cahrens) [inline core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala]: This doesn't use the Option pattern?

Comment (aherbst-broad) [reply core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala]: No, it's a concrete Java object using the builder pattern instead https://cloud.google.com/java/docs/reference/google-cloud-billing/latest/com.google.cloud.billing.v1.ProjectBillingInfo

File: core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala [hunk 2]
```diff
@@ -721,7 +721,9 @@ class HttpGoogleServicesDAO(
...
+            span.putAttribute("billingAccount", AttributeValue.stringAttributeValue(projectBillingInfo.getBillingAccountName))
```
Comment (s-rubenstein) [inline core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala]: Nit: A cleaner thing to do here might be:
```
span.putAttribute("billingAccount", AttributeValue.stringAttributeValue(projectBillingInfo.getBillingAccountName match {
            case value => value
            case null => ""
          }))
```

As opposed to doing the null check.

**Review by s-rubenstein — APPROVED** [review]
LGTM, one nit.

File: core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala (hunk 2 above)
Comment (s-rubenstein) [reply core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala]: You may need to swap the cases or change that slightly, I don't know for sure.

Comment (aherbst-broad) [reply core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala]: I think we need to wrap in an option or use a stable name. I'll wrap in an option

**Pull request merged**